bounties/pip/elementpath/1/vulnerability.json

```diff
@@ -0,0 +1,54 @@
+{
+    "PackageVulnerabilityID": 1,
+    "DisclosureDate": "2020-10-05",
+    "AffectedVersionRange": "*",
+    "Summary": "XML Entity Expansion",
+    "Contributor": {
+        "Discloser": "",
+        "Fixer": ""
+    },
+    "Package": {
+        "Registry": "pip",
+        "Name": "elementpath",
+        "URL": "https://pypi.org/project/elementpath/",
+        "Downloads": "601,388"
+    },
+    "CWEs": [
+        {
+            "ID": "CWE-776",
+            "Description": "Improper Restriction of Recursive Entity References in DTDs ('XML Entity Expansion')"
+        }
+    ],
+    "CVSS":
```

Comment:
Be careful - it seems like you have missed an opening `{`.

Reply:
Oops, thanks for the update. Updated.